src/hotspot/cpu/x86/x86_32.ad

```diff
@@ -1540,37 +1534,49 @@ encode %{
   // adding a syntax that specifies the sizes of fields in an order,
   // so that the adlc can build the emit functions automagically
 
+  // Set instruction mark in MacroAssembler. This is used only in
+  // instructions that emit bytes directly to the CodeBuffer wraped
+  // in the MacroAssembler. Should go away once all "instruct" are
+  // patched to emit bytes only using methods in MacroAssembler.
+  enc_class SetInstMark %{
```

Comment:
Do you have separate RFE for that?

Reply:
Created this one: https://bugs.openjdk.org/browse/JDK-8322876